### PR TITLE
[CI] Fix ownership and parameterize output directory

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -40,10 +41,11 @@ jobs:
       - name: Build website
         run: |
           docker compose run --rm web /bin/bash -c "bundle check || bundle install --jobs=3"
-          docker compose run --rm web make build
+          docker compose run -e O --rm web make build
```
Comment (beta-ziliani) [inline .github/workflows/deploy.yml:44]:
```suggestion
          docker compose run -e $O --rm web make build
```

**Review by beta-ziliani — CHANGES_REQUESTED** [review]
Why "O"?

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (straight-shoota) [reply .github/workflows/deploy.yml:44]: No, that's correct. `-e O` means to pass the environment variable `O` into the container. It's short for `-e O=$O`.

**Comment by straight-shoota** [comment]
`O` is the name for the target directory in `Makefile`. It seems easier to just utilize that name as well outside of it. It's established for that.

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**